draft-ietf-ohai-ohttp.md

```diff
@@ -757,20 +784,20 @@ RECOMMENDED; see {{server-responsibilities}}.
 To achieve the stated privacy goals, the Oblivious Relay Resource cannot be
 operated by the same entity as the Oblivious Gateway Resource. However,
 colocation of the Oblivious Gateway Resource and Target Resource simplifies the
-interactions between those resources without affecting client privacy.
+interactions between those resources without affecting Client privacy.
 
 As a consequence of this configuration, Oblivious HTTP prevents linkability
 described above. Informally, this means:
 
-1. Requests and responses are known only to clients and targets in possession
+1. Requests and responses are known only to Clients and servers in possession
```

Comment:
```suggestion
1. Requests and responses are known only to Clients and Oblivious Gateway Resource in possession
```

Reply:
Isn't it the Target Resource also?  I left this because we defined "server" in this section as being an entity that operates both the gateway and the target.

Reply:
It is, but that's true without OHTTP too, so I think it's only really useful to focus on what the new entities see here.

Reply:
Then maybe ...

```suggestion
1. Requests and responses are known only to Clients and Target Resources, plus Oblivious Gateway Resources that possess
```